src/dataflow/components/nodes/controls/plot-control.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react";
+import Rete from "rete";
+import "./plot-control.sass";
```

Comment:
nit: by convention we have generally been putting the sass import last

Reply:
Fixed.